baselines/webworker.generated.d.ts

```diff
@@ -900,6 +900,13 @@ interface WorkerUtils extends Object, WindowBase64 {
     setTimeout(handler: any, timeout?: any, ...args: any[]): number;
 }
 
+interface Canvas2DContextAttributes {
```

Comment:
hmm. @zhengbli this should not be here. is there a way to exclude it out from one output and not the other?